data/quarg/quarg missions.txt

```diff
@@ -318,7 +316,7 @@ mission "Ask Quarg About Coalition Late"
 	invisible
 	landing
 	source
-		attributes "quarg"
+		attributes "hai quarg" "human quarg"
```

Comment:
Could `Ask Quarg About Coalition Late` remain as is? The idea being that Gohl and the other Quarg who lived on the three rings have been constantly hopping between the other Quarg areas without really ever settling anywhere, so for Gohl in particular I find it thematically fitting that you could encounter it anywhere.